atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -435,7 +435,8 @@ private void sanityCheckRingConsistency() {
         for (InetSocketAddress host : currentPools.keySet()) {
             Cassandra.Client client = null;
             try {
-                client = CassandraClientFactory.getClientInternal(host, config.ssl(), config.socketTimeoutMillis(), config.socketQueryTimeoutMillis());
+                client = CassandraClientFactory.getClientInternal(host, config.credentials(),
+                        config.ssl(), config.socketTimeoutMillis(), config.socketQueryTimeoutMillis());
                 try {
                     client.describe_keyspace(config.keyspace());
                 } catch (NotFoundException e) {
```

Comment:
Not affected by this PR, but this thing feels weird to me.  Is there some side-effect to describe_keyspace that I don't know about?